src/Serilog.Sinks.EventLog/Serilog.Sinks.EventLog.csproj

```diff
@@ -7,7 +7,7 @@
     <NeutralLanguage>en-US</NeutralLanguage>
     <VersionPrefix>3.1.0</VersionPrefix>
     <Authors>Jeremy Clarke;Fabian Wetzel</Authors>
-    <TargetFramework>net45</TargetFramework>
+    <TargetFrameworks>netstandard2.0;</TargetFrameworks>
```

Comment:
Could use `net45;netstandard2.0` here, to keep .NET 4.5 compat?

Reply:
Using `net45;netcoreapp2.0` might be another decent option to consider, since it'd be easier to constrain use/dependency on the compat package.

Reply:
Might need to revert https://github.com/serilog/serilog-sinks-eventlog/pull/25/commits/2fe88dfb6d0f34c6888176774d1399b6802b3f31 since `net45` doesn't support it.

Reply:
We can use #if directives. [There are pre-defined symbols for .NET core and standard](https://docs.microsoft.com/en-us/dotnet/core/tutorials/libraries).

Reply:
Done!